docs/design/autoware-interfaces/components/planning.md

```diff
@@ -1,5 +1,31 @@
+| TrafficLight recognition result | `/perception/traffic_light_recognition/traffic_signals`<br>(_autoware_auto_perception_msgs/msg/TrafficSignalArray_) | This is the real time information about the state of each traffic light. Planning stack will extract the one that is relevant to planned path and use it to decide whether to stop at intersections. |
+
+| Input From Localization Component | Topic Name(Data Type)                                        | Explanation                                                |
+| --------------------------------- | ------------------------------------------------------------ | ---------------------------------------------------------- |
+| Vehicle Kinematics                | `/localization/kinematic_state`<br>(_nav_msgs/msg/Odometry_) | This includes ego vehicle's pose and velocity information. |
+| Vehicle Acceleration              | `/localization/acceleration`<br>(\_)                         | This includes ego vehicle's acceleration.                  |
+
+| Input From Map Component | Topic Name(Data Type)                                             | Explanation                                                                                                                                                                                                                                                                                                         |
+| ------------------------ | ----------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Map data                 | `/map/vector_map`<br>(_autoware_auto_mapping_msgs/msg/HADMapBin_) | This includes all static information about the environment, such as: <ul><li>Lane connection information used for route planning from starting position to goal position</li><li>Lane geometry to generate reference path used to calculate trajectory </li><li> All information related to traffic rules</li></ul> |
+
+| Input From HMI       | Topic Name(Data Type)                                        | Explanation                                                |
+| -------------------- | ------------------------------------------------------------ | ---------------------------------------------------------- |
+| Vehicle Kinematics   | `/localization/kinematic_state`<br>(_nav_msgs/msg/Odometry_) | This includes ego vehicle's pose and velocity information. |
+| Vehicle Acceleration | `/localization/acceleration`<br>(\_)                         | 自車加速度                                                 |
```

Comment:
the data type is missing and the explanation is in Japanese.

Reply:
Oops. I didn't suppose to change the autoware-interface! Sorry.